fe/fe-core/src/main/java/com/starrocks/load/routineload/RoutineLoadJob.java

```diff
@@ -309,7 +309,7 @@ public enum JobSubstate {
     @SerializedName("warehouseId")
     protected long warehouseId = WarehouseManager.DEFAULT_WAREHOUSE_ID;
 
-    // no needs to persist
+    @SerializedName("wcr")
```

Comment:
create an json string without `wcr` in the json, check what the computeResource value will be after deserialize. Could be possibly handle the null/empty value in gsonPostProcess() if needed to avoid unexpected NPE exception.